src/main/java/net/mojodna/osm2orc/standalone/OsmChangesetXml2Orc.java

```diff
@@ -0,0 +1,169 @@
+package net.mojodna.osm2orc.standalone;
+
+
+import de.topobyte.osm4j.core.model.util.OsmModelUtil;
+import net.mojodna.osm2orc.standalone.model.Changeset;
+import net.mojodna.osm2orc.standalone.parser.ChangesetXmlHandler;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.common.type.HiveDecimal;
+import org.apache.hadoop.hive.ql.exec.vector.*;
```

Comment:
Please configure IntelliJ to expand these.

![image](https://cloud.githubusercontent.com/assets/45/21167722/4f588aa2-c163-11e6-98a9-b18d57d2db75.png)

(`999`)